<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>11</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_estoque"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>DESCRICAO</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>PRECO</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_CODIGO"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_preco"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_descricao"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>CODIGO</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>ESTOQUE</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>240</y>
      <width>95</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_incluir">
       <property name="text">
        <string>incluir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_alterar">
       <property name="text">
        <string>alterar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_excluir">
       <property name="text">
        <string>excluir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_listar">
       <property name="text">
        <string>listar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="radioButton_PRODUTO">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PRODUTO</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_CLIENTE">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CLIENTE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_PEDIDO">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PEDIDO</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>231</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>CODIGO</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_CODIGOCLIE"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>NOME</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_NOME"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>ENDERECO</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_ENDERECO"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>TELEFONE</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_TELEFONE"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>CPF</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_CPF"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>137</width>
      <height>98</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_Pedido">
       <property name="text">
        <string>FAZER PEDIDO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_consultarCliente">
       <property name="text">
        <string>CONSULTAR CLIENTE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_consultarPedido">
       <property name="text">
        <string>CONSULTAR PEDIDO</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit_pedido">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_cliente">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
